expression/builtin_encryption.go

```diff
@@ -533,7 +534,26 @@ type builtinRandomBytesSig struct {
 
 // See https://dev.mysql.com/doc/refman/5.7/en/encryption-functions.html#function_random-bytes
 func (b *builtinRandomBytesSig) eval(row []types.Datum) (d types.Datum, err error) {
-	return d, errFunctionNotExists.GenByArgs("RANDOM_BYTES")
+	args, err := b.evalArgs(row)
+	if err != nil {
+		return types.Datum{}, errors.Trace(err)
+	}
+	arg := args[0]
+	if arg.IsNull() {
+		return d, nil
+	}
+	size := arg.GetInt64()
+	if size < 1 || size > 1024 {
```

Comment:
MySQL returns the following error code and error message 
"mysql> select RANDOM_BYTES(2048);
ERROR 1690 (22003): length value is out of range in 'random_bytes'"

Please use standard error code or add a TODO comment here (Then we will fix it latter.).